### PR TITLE
284 ux header missing test cases

### PR DESCRIPTION
**Pull request by scott-dickson**
Please review my tests for ux-header

### CODE REVIEW

File: src/components/ux-header/ux-header.feature [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Feature: ux-header test suite
+
+	Scenario Outline: Loading ux-header
+		Given I have loaded component "ux-header" with use case "<usecase>"
+		Then "<element>" will be visible
+		And the element "<element>" will have the class "<class>"
+		And the element "<element>" should have attribute "id" containing "<idvalue>"
+		And the element "<element>" should have the text "<text>"
+		Examples:
+			| usecase			| element	| class		| idvalue	| text				|
```
Comment (ScottMaclure) [inline src/components/ux-header/ux-header.feature]: Q) pipe alignment?

Comment (scott-dickson) [inline src/components/ux-header/ux-header.feature]: I'm using tabs.  Should I use spaces?

I followed ux-label.feature

https://github.com/ractive-foundation/ractive-foundation/blob/master/src/components/ux-label/ux-label.feature

Comment (ivanwills) [inline src/components/ux-header/ux-header.feature]: Use tab for indentation before any other text and spaces after.

**Comment by scott-dickson** [comment]
Fixed the feature formatting issues.  Thanks Ivan and Scott

**Pull request merged**